src/components/Table/Table.css

```diff
@@ -3,15 +3,6 @@
 .container {
   min-width: 100%;
   clear: both;
-  @media only screen and (max-width: 960px) {
-    overflow-x: scroll;
-  }
-}
-
-.innerContainer {
-  @media only screen and (max-width: 960px) {
-    min-width: 1325px;
-  }
```

Comment:
Do you have a screenshot of what your solution looks like? I don't have my computer on me so I can't see it unless you post a screenshot.